lib/src/layout/sidebar_item.dart

```diff
@@ -0,0 +1,502 @@
+import 'package:flutter/foundation.dart';
+
+import 'package:macos_ui/macos_ui.dart';
+import 'package:macos_ui/src/library.dart';
+import 'package:macos_ui/src/theme/macos_colors.dart';
+
+const Color _transparent = const Color(0x00000000);
+const Duration _kExpand = Duration(milliseconds: 200);
+const ShapeBorder _defaultShape = const RoundedRectangleBorder(
+  borderRadius: const BorderRadius.all(const Radius.circular(7.0)),
+);
+const CupertinoDynamicColor _defaultDisclosureColor =
+    const CupertinoDynamicColor.withBrightness(
+  color: Color(0x1A000000),
+  darkColor: Color(0x42FFFFFF),
+);
+
+Color _textLuminance(Color backgroundColor) {
```

Comment:
This should be made into a global utility for any other widgets that might need it

Reply:
It already is